src/notifications/src/index.ts

```diff
@@ -21,7 +21,11 @@ export default function sendEmail(req: any) {
     
     const email = new Email({

```

Comment:
Maybe move the email code out of index and have index.ts just import NotificationConsumer so that it does the set up for kafka, and you can run using ts-node index.ts instead of NotificationConsumer